poly-commitment/src/ipa.rs

```diff
@@ -730,13 +744,16 @@ impl<G: CommitmentCurve> SRS<G> {
         let d = <G::ScalarField as UniformRand>::rand(rng);
         let r_delta = <G::ScalarField as UniformRand>::rand(rng);
 
-        // delta = (g0 + u*b0)*d + h*r_delta
-        let delta = ((g0.into_group() + (u.mul(b0))).into_affine().mul(d) + self.h.mul(r_delta))
-            .into_affine();
+        // Compute delta, the commitment
+        // delta = G0^d * U_base^{b0*d} * H^r_delta   (as a group element, in multiplicative notation)
```

Comment:
Nit: I would maybe keep the additive notation to avoid confusion?

Reply:
Changed to additive.